routes/todo.js

```diff
@@ -1,8 +1,21 @@
 const { Router } = require("express");
 const { ToDoController } = require("../controllers");
+const authorized = require("../middleware/auth");
 const router = Router();
 
+router.use(authorized());
+
 router.get("/", ToDoController.getAllToDo);
-router.post("/:id/", ToDoController.createToDo);
+router.post("/create", ToDoController.createToDo);
+
+router.route("/:id")
+    .get(ToDoController.getParticularToDo)
+    .put(ToDoController.editToDo)
+    .patch(ToDoController.editToDoPatch)
+    .delete(ToDoController.deleteToDo);
+
+router.post("/:id/add-collaborators", ToDoController.addCollaborator);
```

Comment:
I guess from the code that we are adding one collaborator at a time. So make the `add-collaborators` singular to avoid confusion,

Reply:
sure